lib/KSelect/index.vue

```diff
@@ -43,6 +43,7 @@
           :class="labelClasses"
           :style="activeColorStyle"
         >
+          <!-- @slot Displays the label -->
```

Comment:
```suggestion
          <!-- @slot Optional slot as alternative to `label` prop -->
```

Reply:
I apologize for the delay in addressing the PR comments. The comments have now been addressed. Please review at your earliest convenience.

Reply:
No worries @yeshwanth235. Thank you!! 🚀